### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by katarzyna-z**
Summary of changes:
- Updated examples
- Snap capitalized
- Added link to release page
- Removed SNAP_PATH

How to verify it:
- do steps descripted in example
- read documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,113 +31,109 @@ docker run --name postgres_server -p 5432:5432 --env 'DB_USER=snap' \
...
+database | string | the name of database 
+table_name | string | the name of table
 
-In one terminal window, open the snap daemon :
-```
-$ snapd -l 1 -t 0
-```
+### Examples
 
-In another terminal window:
```
Comment (kindermoumoute) [inline README.md]: typo "and and"

File: README.md [hunk 2]
```diff
@@ -31,113 +31,109 @@ docker run --name postgres_server -p 5432:5432 --env 'DB_USER=snap' \
...
+* systemd: `systemctl start snap-telemetry`
+* command line: `sudo snapd -l 1 -t 0 &`
 
-Load psutil plugin
-```
-$ snapctl plugin load $SNAP_PSUTIL_PLUGIN/build/rootfs/snap-plugin-collector-psutil
-```
 
-See available metrics for your system
+Download and load Snap plugins (paths to binary files for Linux/amd64)::
```
Comment (kindermoumoute) [inline README.md]: typo "::"

File: examples/psutil-postgresql.json [hunk 3]
```diff
@@ -1,37 +1,30 @@
...
+    "type": "simple",
+    "interval": "10s"
+  },
+  "workflow": {
+    "collect": {
+      "metrics": {
+        "/intel/psutil/load/load1": {},
+        "/intel/psutil/load/load15": {}
+      },
+      "config": null,
```
Comment (kindermoumoute) [inline examples/psutil-postgresql.json]: We can remove those fields ("config" and "process") when we don't use it.

File: README.md (hunk 2 above)
Comment (katarzyna-z) [inline README.md]: Corrected

File: README.md (hunk 1 above)
Comment (katarzyna-z) [inline README.md]: Corrected

File: examples/psutil-postgresql.json (hunk 3 above)
Comment (katarzyna-z) [inline examples/psutil-postgresql.json]: I removed `config` and `process` from task.

**Comment by kindermoumoute** [comment]
LGTM

**Pull request merged**